spec/contracts/projects/update_contract_spec.rb

```diff
@@ -41,14 +41,31 @@
       end
     end
 
+    let(:admin_only_custom_field) do
+      build_stubbed(:integer_project_custom_field, admin_only: true).tap do |cf|
+        cf.id = "1002"
+        allow_any_instance_of(CustomValue) # rubocop:disable RSpec/AnyInstance
+          .to receive(:custom_field).and_return(cf)
```

Comment:
This and other identical calls will override each other, so return will depend on evaluation order, probably in `let(:project)` by `not_enabled_custom_field` being last